src/style.css

```
@@ -0,0 +1,44 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 2px;
+}
+
+.stars :first-child {
+  margin-left: 0;
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.stars--5 > .stars--star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 > .stars--star:first-child {
+  background-image: url(images/star-active.svg);
+}
```

it's better to combine them all

```suggestion
.stars--5 > .stars--star,
.stars--1 > .stars--star:first-child {
  background-image: url(images/star-active.svg);
}
```